utils/logalloc.hh

```diff
@@ -208,7 +208,10 @@ class region_group {
         }
     };
 
-    struct on_request_expiry {
+    class on_request_expiry {
+        sstring _name;
+    public:
+        explicit on_request_expiry(sstring name) : _name(name) {}
```

Comment:
`std::move(name)`

Reply:
I removed `name` in favor of static message (if this semaphore is used in one place only, noone would benefit from the ability to give it a generic `name`...)